homework23/task_1.py

```diff
@@ -0,0 +1,31 @@
+class Person :
+    def __init__(self,name,age):
+        self._name = name
+        self._age = age
+    
+    def set_name(self,n):
+        self._name = n
+        
+    def get_name(self):
+        return self._name
+    
+    def set_age(self,a):
+        self._age = a
+    
+    def get_age(self):
+        return self._age
```

Comment:
კარგი იქნება "პიტონური გზით" რომ გადაწერო ეს getters and setters და @property (დეკორატორები) გამოიყენო.